src/coreclr/vm/ClrEtwAll.man

```diff
@@ -7451,8 +7451,8 @@
                              message="$(string.MonoProfilerPublisher.GCAllocationKeywordMessage)" symbol="CLR_MONO_PROFILER_GC_ALLOCATION_KEYWORD" />
                     <keyword name="GCMovesKeyword" mask="0x400000"
                              message="$(string.MonoProfilerPublisher.GCMovesKeywordMessage)" symbol="CLR_MONO_PROFILER_GC_MOVES_KEYWORD" />
-                    <keyword name="GCHeapCollectKeyword" mask="0x800000"
-                             message="$(string.MonoProfilerPublisher.GCHeapCollectKeywordMessage)" symbol="CLR_MONO_PROFILER_GC_HEAPCOLLECT_KEYWORD" />
+                    <keyword name="ManagedHeapCollectKeyword" mask="0x800000"
```

Comment:
Are we also wanting to make this change for the Mono provider?  Seems reasonable, but I want to explicitly ask.

Reply:
Thanks for the catch! It wasn't my intent, I was just doing a find-and-replace.